# Message defining the current state of a moving object

std_msgs/Header header

# Position of the object
geometry_msgs/Point position

# Linear velocity of the object
geometry_msgs/Vector3 velocity

# Angular velocity of the object
geometry_msgs/Vector3 angular_velocity